### DISCUSSION TITLE
Deleting users from ExtPay and preventing access to extension features

### DISCUSSION THREAD

**Discussion by jeffsdevinfo** [Glench/ExtPay#188] [General]
How can I remove registered users from ExtPay in case a user request a refund?  I know I can refund the user's purchase through Stripe through the Stripe dashboard, so that is simple enough but they would still have access to the extension features.

**Comment by connallykendrick** [comment]
Right. In this case we would need to terminate access - rather than allowing the remainder of the previously paid period to run it's course

**Reply by connallykendrick** [reply]
I have no ability to see or remove trial emails currently. I'm in Dev mode, and have used multiple emails for trial testing, but none of those show up on the extension pay website @Glench .

**Comment by Glench** [comment]
Just email me if you want to remove access!
